gatsby-node.js

```diff
@@ -82,7 +82,9 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     }
 
     createPage({
-      path: frontmatter.path || slug,
+      path: frontmatter.path
+        ? path.join(frontmatter.path, '/')
+        : path.join(slug, '/'),
```

Comment:
Are we sure that a `slug` won't already have a trailing slash?

Reply:
i'm pretty sure it won't.

for this line, this is for the `allMdx` nodes, which pulls `slug` off of `fields`. we add that node field for all nodes with type `Mdx` in `onCreateNode` lifecycle in the same file on [line 142](https://github.com/newrelic/developer-website/blob/614ba41a3d133b3728ef299dcf26e574c7741c21/gatsby-node.js#L142). in `createFilePath()` we set the `slug` value with the option `trailingSlash: false`.